tests/pw/pages/productsPage.ts

```
@@ -612,6 +612,24 @@ export class ProductsPage extends AdminPage {
+    // get product edit nonce
+    async getProductEditNonce(): Promise<string> {
+        await this.gotoUntilNetworkidle(data.subUrls.frontend.vDashboard.dashboard);
+        const url = await this.getAttributeValue(selector.vendor.vDashboard.products.addNewProduct, 'href');
+        const nonce = url?.match(/_dokan_edit_product_nonce=([\w\d]+)/)?.[1];
+        if (!nonce) throw new Error('Nonce not found');
+        return nonce;
+    }
+
+    // go to product edit by id
+    async goToProductEditById(productId: string, nonce: string = PRODUCT_EDIT_NONCE): Promise<void> {
+        if (!isNaN(Number(productId))) {
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)